README.md

```diff
@@ -29,6 +29,51 @@ How Do I Get It?
 How Do I Use It?
 ---
 
+### In node (JavaScript):
+
+**Load**
+
+```JavaScript
+var yaml = require('yaml-js');
+
+console.log(yaml.load('--- \n' +
+    'phrase1: \n' +
```

Comment:
This is a little pernickety, especially since there's no style guide, but could you use 2 space indentation so it's consistent with the codebase?